src/Containers/PanelCard/PanelCard.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+import { PanelCard, IPanelCardProps, OperationState } from './PanelCard';
+
+export default {
+    title: 'Components/PanelCard',
+    component: PanelCard,
```

Comment:
argtypes are missing on cancel action()

Reply:
@ralph-dev , I don't fully understand argtypes in storybook

Reply:
Look at Button how it implements actions so you can record button clicks.

ArgTypes is just 

argTypes: {onClickHandler: action("message on click")}

Reply:
> Look at Button how it implements actions so you can record button clicks.
> 
> ArgTypes is just
> 
> argTypes: {onClickHandler: action("message on click")}

I tried but it doesn't work

Reply:
Yes that's all it intends to do, just show you click / responder events that they are working